.github/workflows/docs-preview.yaml

```diff
@@ -17,89 +17,26 @@ permissions:
   security-events: none
   statuses: none
 
-# Cancel in-progress runs for pull requests when developers push
-# additional changes, and serialize builds in branches.
-# https://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-concurrency-to-cancel-any-in-progress-job-or-run
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
```

Comment:
Wouldn't we want to keep this so that it cancels in-progress docs previews when devs push?

Reply:
We don't have to worry about that because this is not deploying or building anything anymore it is just a question of sharing the URL with the sha code on it.

Reply:
Ah okay! Sweet.